src/log/log.go

```diff
@@ -0,0 +1,33 @@
+package log
+
+import (
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"os"
+)
+
+func NewLogger(logFile *os.File, verbose bool) *zap.SugaredLogger {
```

Comment:
Logger vypisuje na Stderr, ... no podrobny vypis aj do temp suboru (cesta sa da nastavit). Ak by to niekomu padalo, tak nam posle podrobny subor a podla toho budeme vediet zdebugovat, kde nastal problem.